frontend/src/Project/EditControls/CellTypeControls/CellTypeUI/ColoredChannelChip.js

```diff
@@ -1,20 +1,31 @@
+import { Grid } from '@mui/material';
 import Chip from '@mui/material/Chip';
 import { useSelector } from '@xstate/react';
 import { useRaw } from '../../../ProjectContext';
+import { hexToRgb } from '../../../service/labels/cellTypesMachine';
 
-function ColoredChannelChip({ matchedChannels, layer }) {
-  const channel = useSelector(layer, (state) => state.context.layer);
+function ColoredChannelChip({ layer, matchedChannels }) {
```

Comment:
Just double-checking that the flipping of the parameter order here is expected

Reply:
I think this doesn't matter in React